vonk/releasenotes/releasenotes.rst

```diff
@@ -10,6 +10,66 @@ Release notes Vonk
    releasenotes_old
    security_notes
 
+.. _vonk_releasenotes_300:
+
+Release 3.0.0
+-------------
+
+Database
+^^^^^^^^
+
+Please also note the changes in :ref:`3.0.0-beta1 <vonk_releasenotes_300-beta1>`
+
+#. SQL Server: SQL script '20190919000000_Cluster_Indexes_On_EntryId.sql' (found in the /data folder of the Vonk distribution) must be applied to existing Vonk SQL databases (both to the admin and to the data repositories) 
+
+   .. attention::
+
+      Vonk 3.0.0 (using SQL server) will not start unless this script has been applied to the databases. Please note that running the script can take considerable time, especially for large databases.
+
+Feature
+^^^^^^^
+#. Information model (= FHIR version) settings
+
+   #. Although Vonk now supports multiple information models (STU3 and R4) simultaneously, an unused model can be disabled (see :ref:`settings_pipeline`)
+   #. You can set the default (or fallback) information model (previously: STU3), which is used when Vonk can not determine the information model from context (see :ref:`information_model`)
+   #. You can map a path or a subdomain to a specific information model (see :ref:`information_model`), mitigating the need to specify it explicitly in a request
+
+#. Vonk now uses Fhir .NET API 1.4.0
```

Comment:
Include a link to the releasenotes of FHIR API 1.4

Reply:
not yet in my branch, but I have used the future reference